src/core/reference.js

```diff
@@ -0,0 +1,321 @@
+/**
+ * file contains few JS library keywords and their examples.
+ */
+/*
+ * @Decleration  Let
```

Comment:
```suggestion
 * @Declaration  Let
```
Small typo throughout. See [this page for spelling](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements).